src/components/ChatApp/Settings/ChatAppTab.react.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import Translate from '../../Translate/Translate.react';
+import Toggle from 'material-ui/Toggle';
+import PropTypes from 'prop-types';
+
+class ChatAppTab extends Component {
```

Comment:
Use functional component here, as we do not need to declare it as a React component